controller/src/main/java/org/jboss/as/controller/persistence/FilePersistenceUtils.java

```diff
@@ -77,9 +77,13 @@ static void copyFile(final File file, final File backup) throws IOException {
         Files.copy(file.toPath(), backup.toPath(), StandardCopyOption.COPY_ATTRIBUTES, StandardCopyOption.REPLACE_EXISTING);
     }
 
-    static void rename(File file, File to) throws IOException {
-        if (!file.renameTo(to) && file.exists()) {
-            copyFile(file, to);
+    static boolean rename(Path from, Path to) {
+        try {
+            Files.move(from, to);
+            return true;
+        } catch (IOException e) {
+            ControllerLogger.ROOT_LOGGER.cannotRename(e, from, to);
+            return false;
```

Comment:
It seems a little strange to have the method be called rename(), but it actually perform a forced move, maybe a boolean param to indicate you really want to force move if the file already exists would be safer?